### ISSUE TITLE
Operation putResourcePolicy is missing in client-schemas

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#1360]
**Describe the bug**
Operation putResourcePolicy is missing in client-schemas

**SDK version number**
```
$ yarn list aws-sdk
└─ aws-sdk@2.693.0
$ yarn list @aws-sdk/client-schemas
└─ @aws-sdk/client-schemas@1.0.0-gamma.3
```

**Is the issue in the browser/Node.js/ReactNative?**
N/A

**To Reproduce (observed behavior)**

<details>
<summary>Code</summary>

```js
const AWS = require("aws-sdk");
const { schemas } = require("@aws-sdk/client-schemas");

(async () => {
  const region = "us-west-2";
  const v2Client = new AWS.Schemas({ region });
  const v3Client = new schemas({ region });

  console.log(v2Client.putResourcePolicy);
  console.log(v3Client.putResourcePolicy);
})();
```

</details>

<details>
<summary>Output</summary>

```console
[Function (anonymous)]
undefined
```

</details>

**Expected behavior**
The operation `putResourcePolicy` should exist on client-schemas

**Additional context**
This issue was noticed while triaging https://github.com/aws/aws-sdk-js/issues/3249

**Comment by trivikr** [comment]
This happens because operation putResourcePolicy is not present in smithy models.

Code for v3: https://github.com/aws/aws-sdk-js-v3/blob/e19fc4feea5268442370af97e266c982beef4210/codegen/sdk-codegen/aws-models/schemas.2019-12-02.json#L1750-L1789

Code for v2: https://github.com/aws/aws-sdk-js/blob/2f0fd56f1c9530c77cd2a59714b91c76a94de76f/apis/schemas-2019-12-02.normal.json#L848-L893

**Comment by trivikr** [comment]
putResourcePolicy is now available in smithy models https://github.com/aws/aws-sdk-js-v3/blob/a37795799ef045d68f8122cc7a15b9f803c03ae9/codegen/sdk-codegen/aws-models/schemas.2019-12-02.json#L1973-L2012

The models were updated in https://github.com/aws/aws-sdk-js-v3/pull/1457 and were released in [v1.0.0-gamma.7](https://github.com/aws/aws-sdk-js-v3/releases/tag/v1.0.0-gamma.7)
